CMakeLists.txt

```diff
@@ -712,7 +712,7 @@ endif()
 find_library(ICONV_LIBRARIES NAMES iconv libiconv libiconv-2 c)
 find_path(ICONV_INCLUDE_DIR NAMES iconv.h)
 
-if (ICONV_LIBRARIES AND ICONV_INCLUDE_DIR)
+if (NOT ANDROID AND ICONV_LIBRARIES AND ICONV_INCLUDE_DIR)
```

Comment:
I'd suggest to only execute the find_library and find_path above if not on android